fastify.js

```diff
@@ -72,18 +65,18 @@ function build (options) {
 
   const trustProxy = options.trustProxy
   const modifyCoreObjects = options.modifyCoreObjects !== false
-  const requestIdHeader = options.requestIdHeader || DEFAULT_REQUEST_ID_HEADER
+  const requestIdHeader = options.requestIdHeader || defaultInitOptions.requestIdHeader
   const querystringParser = options.querystringParser || querystring.parse
   const genReqId = options.genReqId || reqIdGenFactory(requestIdHeader)
-  const bodyLimit = options.bodyLimit || DEFAULT_BODY_LIMIT
+  const bodyLimit = options.bodyLimit || defaultInitOptions.bodyLimit
 
   // Instance Fastify components
   const { logger, hasLogger } = createLogger(options)
   // Default router
   const router = FindMyWay({
     defaultRoute: defaultRoute,
-    ignoreTrailingSlash: options.ignoreTrailingSlash || DEFAULT_IGNORE_TRAILING_SLASH,
-    maxParamLength: options.maxParamLength || DEFAULT_MAX_PARAM_LENGTH,
+    ignoreTrailingSlash: options.ignoreTrailingSlash || defaultInitOptions.ignoreTrailingSlash,
+    maxParamLength: options.maxParamLength || defaultInitOptions.maxParamLength,
     caseSensitive: options.caseSensitive,
```

Comment:
You should add the default option here as well :)

Reply:
when adding the default option here there are 9 tests failing, that's a pretty strange behavior and I was planning on looking at this (this week-end when I will be back home I will have more time to find what's the problem)

For reference here is the log stack of the failing tests when passing `true` as `caseSensitive` default fallback : https://pastebin.com/XJMcsT4r

Reply:
Good catch!
The test that are failing have the `caseSensitive` option equals to `false`, and the `or` is defaulting to the `true` value.
We can leave this as is since `find-my-way` internally defaults to `true` :)